lib/pengine/pe_output.c

```diff
@@ -541,27 +571,35 @@ pe__cluster_options_html(pcmk__output_t *out, va_list args) {
             break;
     }
 
-    if (is_set(data_set->flags, pe_flag_maintenance_mode)) {
-        xmlNodePtr node = pcmk__output_create_xml_node(out, "li");
-
-        pcmk_create_html_node(node, "span", NULL, "bold", "DISABLED");
-        pcmk_create_html_node(node, "span", NULL, NULL,
-                              " (the cluster will not attempt to start, stop, or recover services)");
-    } else {
-        out->list_item(out, NULL, "Resource management enabled");
-    }
-
     return 0;
 }
 
 int
 pe__cluster_options_text(pcmk__output_t *out, va_list args) {
     pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
 
-    if (is_set(data_set->flags, pe_flag_maintenance_mode)) {
-        fprintf(out->dest, "\n              *** Resource management is DISABLED ***");
-        fprintf(out->dest, "\n  The cluster will not attempt to start, stop or recover services");
-        fprintf(out->dest, "\n");
+    out->list_item(out, NULL, "STONITH of failed nodes %s",
+                   is_set(data_set->flags, pe_flag_stonith_enabled) ? "enabled" : "disabled");
+
+    out->list_item(out, NULL, "Cluster is %s",
+                   is_set(data_set->flags, pe_flag_symmetric_cluster) ? "symmetric" : "asymmetric");
+
+    switch (data_set->no_quorum_policy) {
+        case no_quorum_freeze:
+            out->list_item(out, NULL, "No Quorum policy: Freeze resources");
+            break;
+
+        case no_quorum_stop:
+            out->list_item(out, NULL, "No Quorum policy: Stop ALL resources");
+            break;
+
+        case no_quorum_ignore:
+            out->list_item(out, NULL, "No Quorum policy: Ignore");
+            break;
+
+        case no_quorum_suicide:
+            out->list_item(out, NULL, "No Quorum policy: Suicide");
+            break;
```

Comment:
Let's use "No quorum policy" to keep the capitalization consistent with other options